KomaMRIIO/test/runtests.jl

```diff
@@ -0,0 +1,65 @@
+using TestItems, TestItemRunner
+
+@run_package_tests filter=ti->!(:skipci in ti.tags)&&(:io in ti.tags) #verbose=true
+
+@testitem "IO" tags=[:io] begin
+    using Suppressor
+    using KomaMRICore   # For RawAcquisitionData
+
+    # Test Pulseq
+    @testset "Pulseq" begin
+        path = @__DIR__
+        seq = @suppress KomaMRIIO.read_seq(path*"/test_files/epi.seq") #Pulseq v1.4.0, RF arbitrary
+        @test seq.DEF["FileName"] == "epi.seq"
+        @test seq.DEF["PulseqVersion"] ≈ 1004000
+
+        seq = @suppress KomaMRIIO.read_seq(path*"/test_files/spiral.seq") #Pulseq v1.4.0, RF arbitrary
+        @test seq.DEF["FileName"] == "spiral.seq"
+        @test seq.DEF["PulseqVersion"] ≈ 1004000
+
+        seq = @suppress KomaMRIIO.read_seq(path*"/test_files/epi_JEMRIS.seq") #Pulseq v1.2.1
+        @test seq.DEF["FileName"] == "epi_JEMRIS.seq"
+        @test seq.DEF["PulseqVersion"] ≈ 1002001
+
+        seq = @suppress KomaMRIIO.read_seq(path*"/test_files/radial_JEMRIS.seq") #Pulseq v1.2.1
+        @test seq.DEF["FileName"] == "radial_JEMRIS.seq"
+        @test seq.DEF["PulseqVersion"] ≈ 1002001
+
+        # Test ISMRMRD
+        fraw = ISMRMRDFile(path*"/test_files/Koma_signal.mrd")
+        raw = RawAcquisitionData(fraw)
+        @test raw.params["protocolName"] == "epi"
+        @test raw.params["institutionName"] == "Pontificia Universidad Catolica de Chile"
+        @test raw.params["encodedSize"] ≈ [101, 101, 1]
+        @test raw.params["reconSize"] ≈ [102, 102, 1]
+        @test raw.params["patientName"] == "brain2D_axial"
+        @test raw.params["trajectory"] == "other"
+        @test raw.params["systemVendor"] == "KomaMRI.jl"
+
+        # Test KomaMRIIO.signal_to_raw_data
+        signal1 = Vector()
+        for i=1:length(raw.profiles)
+            signal1 = [signal1; raw.profiles[i].data]
+        end
+        rawmrd = KomaMRIIO.signal_to_raw_data(signal1, seq)
+        @test rawmrd.params["institutionName"] == raw.params["institutionName"]
+        io = IOBuffer()
+        show(io, "text/plain", rawmrd)
+        @test occursin("RawAcquisitionData[", String(take!(io)))
+
+    end
+    # Test JEMRIS
+    @testset "JEMRIS" begin
+        path = @__DIR__
+        obj = KomaMRIIO.read_phantom_jemris(path*"/test_files/column1d.h5")
```

Comment:
Why are you doing `KomaMRIIO.`

Reply:
Addressed in [9ca5957](https://github.com/cncastillo/KomaMRI.jl/commit/9ca5957a5b12364daa8b870d4a81f277683cf293)